packages/babel-preset-mc-app/index.js

```diff
@@ -60,18 +60,14 @@ module.exports = function getBabePresetConfigForMcApp(api, opts = {}) {
           // Will use the native built-in instead of trying to polyfill
           // behavior for any plugins that require one.
           ...(opts.runtime !== 'automatic'
-            ? { useBuiltIns: true, pragmaFrag: 'Fragment' }
+            ? {
+                useBuiltIns: true,
+                importSource: '@emotion/core',
+              }
             : {}),
           runtime: opts.runtime || 'classic',
         },
       ],
-      [
-        '@emotion/babel-preset-css-prop',
```

Comment:
If we don't use the new runtime, we should still use this. I suppose we need to make this and the `babel-plugin-emotion` configurable as well?